guides/v2.2/cloud/trouble/robots-sitemap.md

```diff
@@ -13,11 +13,11 @@ Please make sure that "/" is writable by the web-server.
 
 With {{site.data.var.ece}}, you can only write to specific directories, such as `var`, `pub/media`, `pub/static`, or `app/etc`. When you generate the `sitemap.xml` file using the Admin panel, you must specify the `/media/` path.
 
-You do not have to generate a `robots.txt` because it generates on demand and stores the contents in the database. It does not create a file, but you can view the content in your browser with the `<domain.your.project>/robots.txt` file.
+You do not have to generate a `robots.txt` because it is generated on demand and stored in the database. It does not create a file, but you can view the content in your browser with the `<domain.your.project>/robots.txt` file.
 
 This requires ECE-Tools version 2002.0.12 and later with an updated `.magento.app.yaml` file. See an example of these rules in the [magento-cloud repository](https://github.com/magento/magento-cloud/blob/master/.magento.app.yaml#L43-L49).
 
-#### To generate a `sitemap.xml` file in version 2.2 and later
+## To generate a `sitemap.xml` file in version 2.2 and later
```

Comment:
Procedure titles are h4 at the moment.

Reply:
**done**

Reply:
I don't see a change.

Reply:
> I don't see a change.

Could you please clarify the expected result here? Thank you!

Reply:
@hguthrie, `##` means `h4` or I am wrong?

Thanks

Reply:
**fixed**